test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightContextUriParserTests.cs

```diff
@@ -31,21 +31,28 @@ private EdmModel GetModel()
             return model;
         }
 
-        // TODO: Support relative context uri and resolving other relative uris
         [Fact]
-        public void ParseRelativeContextUrlShouldThrowException()
+        public void ParseRelativeContextUrlShouldNotThrowException()
         {
-            string relativeUrl = "$metadata#R";
-            Action parseContextUri = () => ODataJsonLightContextUriParser.Parse(new EdmModel(), relativeUrl, ODataPayloadKind.Unsupported, null, true);
-            parseContextUri.Throws<ODataException>(ErrorStrings.ODataJsonLightContextUriParser_TopLevelContextUrlShouldBeAbsolute(relativeUrl));
+            string relativeUrl = "$metadata#People";
+            Action parseContextUri = () => ODataJsonLightContextUriParser.Parse(this.GetModel(), relativeUrl, ODataPayloadKind.Unsupported, null, true, true, new Uri("https://www.example.com/api"));
+            parseContextUri.DoesNotThrow();
+        }
+
+        [Fact]
+        public void ParseRelativeContextUrlWithOnlyDeltaSegmentShouldNotThrowException()
+        {
+            string relativeUrl = "#$delta";
+            Action parseContextUri = () => ODataJsonLightContextUriParser.Parse(this.GetModel(), relativeUrl, ODataPayloadKind.Unsupported, null, true, true, new Uri("https://www.example.com/api"), "People");
+            parseContextUri.DoesNotThrow();
         }
 
 
 #if NETCOREAPP2_0
         [Fact]
         public void ParseContextUrlWithEscapedSpecailMeaningCharactersShouldSucceed()
         {
-            string urlWithUnescapedSpecialMeaningCharacters = "https://www.example.com/api/$metadata#people('i%3A0%23.f%7Cmembership%7Cexample%40example.org')/Dogs";
+            string urlWithUnescapedSpecialMeaningCharacters = "$metadata#people('i%3A0%23.f%7Cmembership%7Cexample%40example.org')/Dogs";

```

Comment:
Why change this to test the relative URL scenario? Instead you should parameterize this test such that it covers both the scenarios where the URL is absolute and relative...

Reply:
I don't remember making this change. Let me fix it.